zipline-security/src/jsMain/kotlin/app/cash/zipline/security/ZiplineSecurity.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2024 Cash App
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.security
+
+import app.cash.zipline.Zipline
+
+/** Guest-side API to Zipline's security services. */
+class ZiplineSecurity(zipline: Zipline) {
```

Comment:
Don't think this is the API we should offer. The `take` is hidden internally meaning I cannot have two unrelated parts of my codebase instantiate this type.

Can `nextBytes` and `nextLong` be extension functions on the service type which is taken directly by user code instead?

Reply:
I *think* that's fine and don't have strong opinions myself. What you suggest certainly sounds like the idiomatic way to expose this in zipline, i think (I'm still learning). Maybe this is an abstraction that belongs in consumer code (higher level)

Reply:
Lets land this as-is and then @JakeWharton  and I can figure out a good pattern for end-users in follow-up?